src/system-module.adoc

```diff
@@ -0,0 +1,257 @@
+[[sec:sysmod]]
+== The System Module
+
+include::styles.adoc[]
+
+The symbols and macros of the system module `$ion` is available everywhere within an Ion document,
+with the version of that module being determined by the spec-version of each segment.
+
+The specific system symbols are largely uninteresting to users since they don't affect the text
+content that they interact with.
```

Comment:
Do you mean users have written `$4` rather than `name`?

Would you like more or different text here?  I admit its not the most clear sentence around...

Reply:
> Do you mean users have written $4 rather than name?

No, I just mean that `name` appears as a field name frequently and so gets encoded as `$4`. It doesn't change the thrust of what you're saying, it's just the most common way for users to interact with the system symbol table apart from LSTs.

> Would you like more or different text here? I admit its not the most clear sentence around...

Maybe:
```suggestion
The specific system symbols are largely uninteresting to users; while the binary encoding heavily leverages the system symbol table, the text encoding that users typically interact with does not.
```
?

Reply:
That's great! Thanks!